sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline.ipynb

```diff
@@ -0,0 +1,423 @@
+    "                if i == 0 or len(line.strip()) == 0:\n",
+    "                    continue\n",
+    "                line_split = line.strip().split(\",\")\n",
+    "                if len(line_split) != 2:\n",
+    "                    print(\"Skipping the invalid line: {}\".format(line))\n",
+    "                    continue\n",
+    "                json_line = dict(json_line_sample)\n",
+    "                json_line[\"image_url\"] += f\"images/{line_split[0]}\"\n",
+    "                json_line[\"label\"] = line_split[1].strip().split(\" \")\n",
+    "\n",
+    "                if i % train_validation_ratio == 0:\n",
+    "                    # validation annotation\n",
+    "                    validation_f.write(json.dumps(json_line) + \"\\n\")\n",
+    "                else:\n",
+    "                    # train annotation\n",
+    "                    train_f.write(json.dumps(json_line) + \"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Basic pipeline job with Image Classification Multilabel task\n",
+    "\n",
+    "## 2.1 Build pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note that the used docker image doesn't suit for all size of gpu compute. Please use the following command to create gpu compute if experiment failed\n",
+    "# !az ml compute create -n gpu-cluster --type amlcompute --min-instances 0 --max-instances 4 --size Standard_NC12"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@pipeline(\n",
+    "    description=\"AutoML Image Clasiification Multilabel Pipeline\",\n",
+    "    default_compute=\"gpu-cluster\",\n",
+    ")\n",
+    "def automl_image_classification_miltilabel(\n",
+    "    image_classification_multilabel_train_data,\n",
+    "    image_classification_multilabel_validation_data\n",
+    "):\n",
+    "    # define the automl image_classification_multilabel task with automl function\n",
+    "    image_classification_multilabel_node = image_classification_multilabel(\n",
+    "        training_data=image_classification_multilabel_train_data,\n",
+    "        validation_data=image_classification_multilabel_validation_data,\n",
+    "        target_column_name=\"label\",\n",
+    "        primary_metric=\"iou\",\n",
+    "        properties={\"_automl_internal_label\": \"latest\",},\n",
+    "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes \n",
+    "        outputs={\"best_model\": Output(type=\"mlflow_model\")},\n",
+    "    )\n",
+    "    image_classification_multilabel_node.set_limits(timeout_minutes=120)\n",
+    "\n",
+    "    image_classification_multilabel_node.extend_search_space(\n",
+    "        [\n",
+    "            ImageClassificationSearchSpace(\n",
```

Comment:
Please use the same search space as used in cli examples

Reply:
Done